### ISSUE TITLE
Why fucking Termius hosts can't export to file!

### ISSUE THREAD

**Issue by tri5m** [termius/termius-cli#198]
Why fucking Termius hosts can't export to file!
Why fucking Termius hosts can't export to file!
Why fucking Termius hosts can't export to file!
Why fucking Termius hosts can't export to file!
Why fucking Termius hosts can't export to file!
Why fucking Termius hosts can't export to file!
Why fucking Termius hosts can't export to file!

**Comment by tri5m** [comment]
<img width="383" alt="image" src="https://user-images.githubusercontent.com/49668053/207532290-3ba2f7bd-873f-41b4-8779-ef7388e3a4e4.png">

**Comment by rmatte** [comment]
You really think that asking rudely like this is going to get this looked at? You need to work on your manners.

**Comment by tri5m** [comment]
> You really think that asking rudely like this is going to get this looked at? You need to work on your manners.

I don't need others to see me.  I'm just venting my anger.  Why can most SSH tools on the market export configurations, but not Termius?  Is it because of your greed?  Your thirst for money?  Or is it to restrict your users from using Chinese applications?  This has resulted in my inability to retrieve and migrate a large number of SSH connection passwords, which is due to your greed.

**Comment by T0sng** [comment]
what？ Not everyone is obliged to provide services to you for free。If you feel that the charged service is unreasonable, you can choose to give up at the beginning.Instead of asking others why they pay after enjoying the paid service。:)

**Comment by T0sng** [comment]
Please don't lower the quality of Chinese people。Where is the application restricting Chinese users?😅😅😅Are you a giant baby? You can choose to go crazy in the mirror and no one tells you, not here. Your behavior and urinating in public places and then telling others that you didn't let others see you.  😄so funny

**Comment by tri5m** [comment]
I am a paying user of this application. I need to export my data when switching to other programs. Most open source applications can export data of one or more structures. On the contrary, this software prohibits users from exporting data in order to get users. What's the difference between this software and those companies that harvest users in China?

**Comment by T0sng** [comment]
Sorry. Looks like I misunderstood, you can go to twitter and search for the name of the software. The first one is their customer service account. I have seen many times that customer service replies to customer-related questions on Twitter. I hope it can help you

**Comment by tri5m** [comment]
We firmly oppose this Chinese-style attitude towards users. This is the spirit of our open source community. Since it is commercial software, don't sell dog meat.

**Comment by Kurasami** [comment]
> Why fucking Termius hosts can't export to file! Why fucking Termius hosts can't export to file! Why fucking Termius hosts can't export to file! Why fucking Termius hosts can't export to file! Why fucking Termius hosts can't export to file! Why fucking Termius hosts can't export to file! Why fucking Termius hosts can't export to file!

Yo, before opening this kind of ticket and with this sort of attitude you should use something like [DevRant](https://devrant.com/) if you want flip off about something. Github is where adults speak to, and respect each other like other adults.

Honestly your attitude is reprehensible and if you were one of my employees and someone sent me this comment. I'd fire you on the spot.

**Comment by Kurasami** [comment]
@EvgeneOskin would you mind closing this?

**Comment by xhzco** [comment]
I have a version that can export configurations, my email is: iamxg@qq.com

<img src=https://github.com/termius/termius-cli/assets/99109879/9a96dfab-c713-434d-bb66-211cdc506672 width=500px />

<img src=https://github.com/termius/termius-cli/assets/99109879/3277ee89-425f-4ae9-8d43-02250abc06d3 width=500px />

<img src=https://github.com/termius/termius-cli/assets/99109879/d16b5746-bbe4-469b-9537-1e2eb86aeef6 width=500px />

**Comment by Scotspaul** [comment]
I agree that we cant export our hosts, i wanted to try the new Beta but not if i have to manually add each host again.
Yes he asked in a terrible manner but he is only saying what others are thinking.
Termius fix this please so ppl will continue to use your products

**Comment by 825i** [comment]
I simply stopped using Termius. Not only does it not do anything special (least of all worth paying for), there has been basically no development on the software either. The last commit here was THREE YEARS AGO.

Stuff like this is completely anti-consumer and it is ridiculous that this would be open source but then have "antifeatures" like this.

Given that they have also deleted some comments here (and will likely delete this). I couldn't care less.

The final straw maybe was that their "new encryption" is only available to paying members. Whilst free users don't get the security that exists, just because they don't pay. Yeah, no thanks.

I highly recommend instead using: https://github.com/xpipe-io/xpipe

Bye.
